### PR TITLE
[CSI v2.6.0] Helm changes  to support CSI 2.6.0 Release

### PR DESCRIPTION
**Pull request by sanchitatntnx**
Added support to specify/modify the squash type for Nutanix Files Storage and Nutanix Dynamic Files Storage in helm values.

### CODE REVIEW

File: charts/nutanix-csi-storage/README.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ The following table lists the configurable parameters of the Nutanix-CSI chart a
 | `fileClass`                   | Activate Nutanix Files Storage Class                                                        | `false`                |
 | `fileClassName`               | Name of the Nutanix Files Storage Class                                                     | `nutanix-file`         |
 | `fileClassRetention`          | Retention policy for the Files Storage Class (Delete, Retain)                               | `Delete`               |
+| `dynFileSquashType`           | Squash type for Nutanix Dynamic File Storage (none, root-squash, all-squash)                | `root-squash`          |
```
Comment (subodh01) [inline charts/nutanix-csi-storage/README.md:121]: Please rename it to dynamicFileSquashType to keep it consistent with other parameters.

File: charts/nutanix-csi-storage/values.yaml [hunk 2]
```diff
@@ -86,7 +86,9 @@ filePath: share
 # Dynamic Files Settings
 #
 fileServerName: file
-
```
Comment (subodh01) [inline charts/nutanix-csi-storage/values.yaml:89]: Please add this blank line back.

**Review by tuxtof — CHANGES_REQUESTED** [review]
@sanchitatntnx let's also expose the option in the `question.yaml` file

we also need to update image to 2.6.0
and check sidecar upgrade

File: charts/nutanix-csi-storage/README.md (hunk 1 above)
Comment (sanchitatntnx) [reply charts/nutanix-csi-storage/README.md]: Done

File: charts/nutanix-csi-storage/questions.yml [hunk 3]
```diff
@@ -121,3 +121,10 @@ questions:
     description: "Specify Nutanix Files server name"
     group: "Nutanix Files Settings"
     show_if: "dynamicFileClass=true"
+  - variable: dynamicFileSquashType
+    label: "Squash Type for NFS"
+    type: string
```
Comment (tuxtof) [inline charts/nutanix-csi-storage/questions.yml:126]: best here is to give a `type: enum` and to limit the choice to supported option

something like

```
- variable: dynamicFileSquashType
  label: "Squash Type for Dynamic NFS"
  type: enum
  default: "root-squash"
  options: ["root-squash", "all-squash", "none"]
  description: "Specify Nutanix Files Squash Type"
  group: "Nutanix Files Settings"
  show_if: "dynamicFileClass=true"
```

**Review by tuxtof — CHANGES_REQUESTED** [review]

File: charts/nutanix-csi-storage/questions.yml (hunk 3 above)
Comment (sanchitatntnx) [reply charts/nutanix-csi-storage/questions.yml]: Done

**Comment by tuxtof** [comment]
now you need to update image version and also sidecar version

**Review by tuxtof — CHANGES_REQUESTED** [review]
update k8s.gcr.io/sig-storage/csi-resizer to 1.6.0

**Review by tuxtof — APPROVED** [review]

**Pull request merged**